streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalStateManagerImplTest.java

```diff
@@ -152,28 +150,6 @@ public void before() {
         checkpointFile = new File(stateManager.baseDir(), StateManagerUtil.CHECKPOINT_FILE_NAME);
     }
 
-    @After
-    public void after() throws IOException {
-        stateDirectory.unlockGlobalState();
-    }
-
-    @Test
-    public void shouldLockGlobalStateDirectory() {
```

Comment:
Should we be verifying that the dir gets deleted in some new tests here?

Reply:
The global directory should _not_ be deleted, just locked/unlocked (until now). We do have a test for the task directory deletion though, in case that's what you were referring to